todo_app/data/Item.py

```diff
@@ -0,0 +1,17 @@
+
+from datetime import datetime
+
+class Item:
+    def __init__(self, id, name, status, desc, due):
+        self.id = id
+        self.name = name
+        self.status = status
+        self.desc = desc
+        self.due = due
+
+    @classmethod
+    def from_trello_card(cls, card, list):
+        due = card['due']
+        if(due != None):
+            due = due.split("T")[0]
```

Comment:
As I mentioned in another comment it would be nice to parse the "due" as a datetime at this point rather than just doing string manipulation.

It will make it easy to perform date arithmetic like calculating "x days remaining" if you have a `date` or `datetime` object. And you can easily change your mind about how to format it for display.

Plus it makes it easier to understand the code to see a parsing a string with a particular format instead of splitting a string and taking the first part.